pkg/cmd/server/api/v1/types.go

```diff
@@ -338,6 +338,28 @@ type ImagePolicyConfig struct {
 	// MaxScheduledImageImportsPerMinute is the maximum number of scheduled image streams that will be imported in the
 	// background per minute. The default value is 60. Set to -1 for unlimited.
 	MaxScheduledImageImportsPerMinute int `json:"maxScheduledImageImportsPerMinute"`
+	// AllowedRegistriesForImport is a white list of registries that we allow to import the
+	// images from.
+	// When a new master config is being generated, several public registries like
+	// docker.io, Red Hat registry, gcr.io, Amazon ECS, quay.io are allowed by default.
+	// If you want to import images from your private Docker Registry, you have to add it into
+	// the list.
+	// If not specified in master config all registries are allowed by default (legacy
```

Comment:
This needs to be cleaned up a bit.  It's not legacy behavior, really.  We'll need to check what registries people are using to make sure we haven't missed any common ones.

Mention here the thing I asked about below - anyone who can create images directly or post ISM can always import images (bypass the validation check).